dsl/src/test/java/org/mule/test/config/ast/ArtifactAstValidationsTestCase.java

```diff
@@ -53,6 +84,65 @@
 @Story(DSL_VALIDATION_STORY)
 public class ArtifactAstValidationsTestCase extends AbstractMuleContextTestCase {
 
+  private Injector injector;
+
+  @Rule
+  public SystemProperty systemProperty = new SystemProperty(ENFORCE_EXPRESSION_VALIDATION_PROPERTY, "true");
+
+  class BasicModule extends AbstractModule {
```

Comment:
the ides of this code apart from testing is that other teams use it as example reference, please add comments